Assets/Scripts/Modules/Stats/IStatModule.cs

```diff
@@ -5,4 +5,5 @@
 
 public interface IStatModule {
 
+	void AssignTitle ();
```

Comment:
What is the functionality of this method? I would assume it's taking in a map tile and giving the unit that tile as their location?